### PR TITLE
[FIX] Fixed menu looks buggy when scrolling (Podio bug_25)

### PR DESCRIPTION
**Pull request by vhusaruk92**

### CODE REVIEW

File: dev/js/main/header/header.js [hunk 1]
```diff
@@ -55,8 +55,9 @@ jQuery(function($) {
         }
     });
 
-    $(window).on('orientationchange',function() {
+    $(window).on('resize',function() {
         cleanup();
+        console.log("vika");
```
Comment (dmh) [inline dev/js/main/header/header.js]: ?

File: dev/js/main/header/header.js [hunk 2]
```diff
@@ -79,8 +80,8 @@ jQuery(function($) {
 
     if (navbar.length) {
         var offsetTop = navbar.offset().top;
-        $(window).on('orientationchange',function() {
-            if ($(window).width() > 992 && touchSupport) {
+        $(window).on('resize',function() {
```
Comment (dmh) [inline dev/js/main/header/header.js]: we should try to fix it without resize event if it possible?

**Review by dmh — CHANGES_REQUESTED** [review]

File: dev/js/main/header/header.js (hunk 2 above)
Comment (vhusaruk92) [reply dev/js/main/header/header.js]: please, check new commit

**Comment by pxamike** [comment]
My pull-request ( https://github.com/t3kit/felayout_t3kit/pull/52 ) should be the proper fix for this issue, so I would not merge this. (However my solution is inspired by this, so cheers to vhusaruk92!)

**Pull request closed without merging**